models/reviews-models.js

```diff
@@ -30,3 +31,17 @@ exports.fetchReviewById = (review_id) => {
     }
   });
 };
+
+exports.fetchReviewComments = (review_id) => {
+  let queryStr = "SELECT * FROM comments";
+  const queryParams = [];
+
+  if (review_id !== undefined) {
```

Comment:
Im gonna be harsh and say this feels unnecessary. If we wanted the review_id to be optional, like in the request body then this might make sense or for re-using in another endpoint. However as this won't be the case it just adds not needed complexity to our code. Would suggest the same for the fetchReviewById model